statwrap/ux.py

```diff
@@ -40,20 +33,25 @@ class DataUploadWidget:
     >>>     uploader = DataUploadWidget('df3', auto_display=False)
     
     >>> display(output)
-    >>> display(uploader.uploader, uploader.submit_button)
+    >>> display(uploader)
     """
 
-    def __init__(self, variable_name, accept: str = '.csv,.xls,.xlsx,.xlsm,.xlsb,.odf,.ods,.odt', auto_display: bool = True):
-        self.accept = accept
+    def __init__(self, variable_name: str = 'df', auto_display: bool = True):
```

Comment:
The docstrings need some rearrangement. 

The parameters should be documented under the __init__ constructor and the attributes and methods should be documented under the class definition. Should be an easy chatgpt copy and paste task. 

See here https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
Think I did this but not sure if I understood

Reply:
At line 47 in ux.py, there is no docstring for the constructor, but there should be, and this should be what documents the parameters. The attributes and some other pieces of the docstring currently on lines 9-43 can stay, and others belong under the init constructor. Here's what ChatGPT suggests. I can dig a little more on this or move this to a new PR if you prefer. 

```
class DataUploadWidget(widgets.VBox):
    """
    A widget for uploading data files and creating a pandas DataFrame.

    Attributes
    ----------
    uploader : widgets.FileUpload
        Widget for file upload.
    submit_button : widgets.Button
        Button to submit the file upload and create the DataFrame.
    variable_name : str
        The name of the variable to store the DataFrame in the IPython environment.

    Examples
    --------
    Single DataFrame Usage:
    >>> uploader = DataUploadWidget("df")
    
    Multiple DataFrame Usage:
    >>> uploader1 = DataUploadWidget("df1")
    >>> uploader2 = DataUploadWidget("df2")
    
    Delayed Output Usage:
    >>> output = widgets.Output()
    >>> with output:
    >>>     uploader = DataUploadWidget('df3', auto_display=False)
    >>> display(output)
    >>> display(uploader)
    """

    def __init__(self, variable_name: str = 'df'):
        """
        Initialize the DataUploadWidget.

        Parameters
        ----------
        variable_name : str, optional
            The name of the variable to store the DataFrame in the IPython environment. Default is 'df'.
        """
        super().__init__()
```